src/holofuel/pages/TransactionHistory/TransactionHistory.test.js

```diff
@@ -79,15 +79,15 @@ describe('TransactionsHistory', () => {
           rows.forEach(async (row, rowIndex) => {
             const transaction = fullRowContent[rowIndex]
             const { getByTestId } = within(row)
-            const whois = await HoloFuelDnaInterface.user.getCounterparty({ agentId: fullRowContent[rowIndex].counterparty })
+            const whois = await HoloFuelDnaInterface.user.getCounterparty({ agentId: fullRowContent[rowIndex].counterparty.id })
             const notesDisplay = transaction.notes === null ? 'none' : transaction.notes
             const dateDisplay = presentDateAndTime(transaction.timestamp).date
             const timeDisplay = presentDateAndTime(transaction.timestamp).time
             const amountToMatch = transaction.direction === 'outgoing' ? `(${presentHolofuelAmount(transaction.amount)})` : presentHolofuelAmount(transaction.amount)
 
             expect(within(getByTestId('cell-date')).getByText(dateDisplay)).toBeInTheDocument()
             expect(within(getByTestId('cell-time')).getByText(timeDisplay)).toBeInTheDocument()
-            expect(within(getByTestId('cell-counterparty')).getByText(whois.nickname) || within(getByTestId('cell-counterparty')).getByText(presentAgentId(transaction.counterparty))).toBeInTheDocument()
+            expect(within(getByTestId('cell-counterparty')).getByText(whois.nickname) || within(getByTestId('cell-counterparty')).getByText(presentAgentId(transaction.counterparty.id))).toBeInTheDocument()
```

Comment:
let's be specific about what we're testing here. Can we just test for `nickname` and not for `id`?